debian/control

```diff
@@ -5,7 +5,7 @@ Maintainer: Jellyfin Packaging Team <packaging@jellyfin.org>
 Uploaders: Jellyfin Packaging Team <packaging@jellyfin.org>
 Rules-Requires-Root: no
 Homepage: https://ffmpeg.org/
-Standards-Version: 5.1.1
```

Comment:
Is this intentional?

Reply:
Yes. `Standards-Version` should be these: https://www.debian.org/doc/debian-policy/upgrading-checklist.html